eval-jam/package.json

```diff
@@ -27,15 +28,17 @@
     "get-repository-url": "^2.0.0",
     "graphviz": "0.0.9",
     "is-builtin-module": "^3.0.0",
+    "jam": "file:../ISSTA-2021-Paper-156",
```

Comment:
I think a lot of these dependencies are only needed because we directly included the Jam sources in our build. Since that appears to be fixed by having Jam as a regular dependency, are they still needed? I guess we need them because the Jam source are still transpiled as part of our build here.

Reply:
Yes, I tried without them but they are runtime dependencies rather than dev dependencies. I can make that more clear.